### PR TITLE
"Death gasp" for a moth

### PR DESCRIPTION
**Pull request by ImRenQ**
## About the PR
added "death gasp" sound for moths

## Why / Balance
https://github.com/space-wizards/space-station-14/pull/27736

## Media

https://github.com/space-wizards/space-station-14/assets/164364533/11195d09-039d-49f2-8fe7-5542c16630e3

https://github.com/space-wizards/space-station-14/assets/164364533/4e312fb7-019f-4893-83e3-5ccac42b859a

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: RenQ
- add: The moths now have their own "death grasp"

### CODE REVIEW

File: Resources/Audio/Effects/Gasp/attributions.yml [hunk 1]
```diff
@@ -24,3 +24,9 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from tgstation at https://github.com/tgstation/tgstation/commit/f7a49c4068f1277e6857baf0892d355f1c055974"
   source: "https://github.com/tgstation/tgstation/tree/f7a49c4068f1277e6857baf0892d355f1c055974/sound/voice/human"
+
+- files:
+  - moth_DeathGasp.ogg
+  license: "CC 0 license"
```
Comment (ImRenQ) [inline Resources/Audio/Effects/Gasp/attributions.yml:30]: How can I fix this?

Comment (Tayrtahn) [reply Resources/Audio/Effects/Gasp/attributions.yml:30]: You can see the valid license identifiers here: https://github.com/space-wizards/RobustToolbox/blob/master/Schemas%2Frga_validators.py

**Comment by Tayrtahn** [comment]
Are you sure about that license? Looks like other stuff from tgstation is licensed under CC-BY-SA-3.0

**Comment by ImRenQ** [comment]
> Are you sure about that license? Looks like other stuff from tgstation is licensed under CC-BY-SA-3.0

I don't know. in this case, can CC-BY-SA-3.0 be used?
изображение

**Comment by deltanedas** [comment]
if you use the original from freesound it can be CC0-1.0 otherwise CC-BY-SA-3.0 with tgs modified version

**Comment by ImRenQ** [comment]
thanks

File: Resources/Prototypes/SoundCollections/deathgasp.yml [hunk 2]
```diff
@@ -21,3 +21,8 @@
   files:
   - /Audio/Effects/Gasp/deathgasp_1.ogg
   - /Audio/Effects/Gasp/deathgasp_2.ogg
+
+- type: soundCollection
+  id: moth_DeathGasp
```
Comment (AJCM-git) [inline Resources/Prototypes/SoundCollections/deathgasp.yml:26]: last one: prototype Ids should be PascalCase (so MothDeathGasp in this case)

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**